tests/test-integration/src/test/java/org/eclipse/jetty/test/AllowedResourceAliasCheckerTest.java

```diff
@@ -76,46 +67,39 @@ public static void beforeAll() throws Exception
         _context.addServlet(DefaultServlet.class, "/");
         _server.setHandler(_context);
 
-        _baseDir = getResourceDir().resolve("baseDir").toFile();
-        _baseDir.deleteOnExit();
-        assertFalse(_baseDir.exists());
-        _context.setResourceBase(_baseDir.getAbsolutePath());
-    }
-
-    @AfterAll
-    public static void afterAll() throws Exception
-    {
-        _client.stop();
-        _server.stop();
+        _baseDir = workDir.getEmptyPathDir().resolve("baseDir");
+        _context.setBaseResource(new PathResource(_baseDir));
     }
 
     @AfterEach
-    public void afterEach()
+    public void afterAll() throws Exception
```

Comment:
Rename to `after()` or `dispose()`.

Reply:
this one is currently called `public void teardown()`

Reply:
now called `public void dispose()`